packages/office-ui-fabric-react/src/components/Announced/docs/AnnouncedOverview.md

```diff
@@ -2,15 +2,12 @@ The Announced component aims to fill several of the accessibility gaps that exis
 It provides text for the screen reader in certain scenarios that are lacking comprehensive updates, particularly those showing
 the completion status or progress of operation(s).
 
-Some real-world applications of the component include copying, uploading, or moving many files, deleting or renaming a single file,
-"lazy loading" of page sections that do not appear all at once, and appearance of search results.
+Some real-world applications of the component include copying, uploading, or moving many files; deleting or renaming a single file;
+"lazy loading" of page sections that do not appear all at once; and appearance of search results.
 
-The Announced component currently has the following 4 documented use cases:
+The Announced component currently has the following documented use cases:
 
 1. [Quick Actions](#/examples/announced/quickactions): Operations such as editing text or deletion that are short enough that they do not require a status during progress.
-
```

Comment:
Should we keep the spacing here? Without it, it looks a little cramped.

![image](https://user-images.githubusercontent.com/14134000/55990086-a87c2b80-5c5b-11e9-9ce8-e19deae03c04.png)

vs. 

![image](https://user-images.githubusercontent.com/14134000/55990055-900c1100-5c5b-11e9-972d-390acd287c56.png)

@betrue-final-final what do you think?

Reply:
I wouldn't. that looks excessive and it oddly is harder to follow the bullets with so much space.

Reply:
okay, looks fine then!

Reply:
I think the ideal for bullet point spacing would be a bit more than there is now, but less than there is if you put an extra line between each point in the markdown.